gulp-core-build-typescript/src/RunApiExtractorOnExternalApiTypes.ts

```diff
@@ -0,0 +1,37 @@
+import * as gulp from 'gulp';
+import * as path from 'path';
+import * as fs from 'fs';
+
+import { GulpTask } from '@microsoft/gulp-core-build';
+import { ExternalApiHelper } from '@microsoft/api-extractor';
+
+const files: string[] = ['external-api-types/es6-collections/index.d.ts',
```

Comment:
The top-level folders for our NPM packages are always standard roles such as "src", "temp", "typings", etc.

To solve this problem, maybe we coudl define a "resources" folder that would contain other build inputs that are not source code.  So the path name would be something like:

./resources/external-types/es6-collections